llpc/test/shaderdb/partial_pipeline_cache_test/pipelines/PipelineVsFs_ConstantData_Vs1Fs1.pipe

```diff
@@ -0,0 +1,84 @@
+; Test that constant data in the vertex shader is handled correctly.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s
```

Comment:
Can you check for `AMDLLPC SUCCESS`?

Reply:
It seems that `llvm-objdump` will not generate `AMDLLPC SUCCESS`.

Reply:
OK, this is fine as-is.